11ty/definitions/adhd.md

```diff
@@ -0,0 +1,30 @@
+---
+title: ADD/ADHD
+slug: adhd
+defined: true
+speech: noun
+reading:
+- text: "What Is ADHD? Meaning, Symptoms & Tests"
+  href: https://www.additudemag.com/what-is-adhd-symptoms-causes-treatments/
+- text: "No Time To Confront Racism In Neurodiversity"
+  href: https://blackgirllostkeys.com/adhd/no-time-to-confront-racism-in-neurodiversity/
+- text: DSM-5 ADHD Diagnosis
+  href: https://www.cdc.gov/ncbddd/adhd/diagnosis.html
+
+---
+
+ADHD is a neurological disorder that affects someone's ability to focus or execute tasks. It is commonly thought of as "the hyper kid in class", though it is much more complex and nuanced.
+
+## Common Usage
+
+People often say "I'm so ADD/ADHD" when they're referring to short periods of hyperactivity or inability to focus. It's used colloquially much like "I'm so OCD," and has many of the same negative ramifications for neurodivergent folks.
+
+When people think of ADHD, they think of a very specific character. This character is almost always a boy, and he is usually hyperactive and extremely distractible.
+
+## Impact
+
+What used to be diagnosed as either ADD or ADHD is now a single diagnosis called ADHD. ADHD has a few distinct subtypes: Inattentive, Hyperactive, and combined type. The long-time emphasis on hyperactivity has resulted in a lot of gender and racial inequity in ADHD diagnosis and treatment. To parahprase Rene(TODO add accent to e) Brooks, it's very difficult for women, especially black women, to obtain a proper diagnosis and treatment for their ADHD.
+
+The casual usage of ADHD to refer to occasional focus or hyperactivity issues creates the stigma that ADHD is a "fake" mental state that you grow out of, rather than a chronic neurological disorder that requires treatment. As is the case with many disorders, the symptoms and behaviors happen to most people - what results in a diagnosis is when those symptoms create a significant impairment in someone's ability to function on a regular basis.
```

Comment:
```suggestion
The casual usage of ADHD to refer to occasional focus or hyperactivity issues creates the stigma that ADHD is a "fixable", "temporary", or "fake" mental state that you grow out of, rather than a chronic neurological disorder that can involve medical treatment. As is the case with many disorders, the symptoms and behaviors can happen to many people regardless of diagnosis. What results in a diagnosis is when those symptoms create a significant impairment in someone's ability to function on a regular basis and they have access and means to obtaining that care.
```